content/21.pharmaceuticals-app.md

```diff
@@ -59,7 +59,8 @@ The sample size for this study was very small, with 29 total patients enrolled,
 The second study [@doi:10.1007/s00705-021-04956-9] was larger, with 96 patients enrolled, and included only individuals with mild to moderate symptoms who were randomized into two groups: one receiving chloroquine (CQ) in addition to SOC, and the other receiving favipiravir in addition to SOC.
 This study reported a non-significant trend for patients receiving favipiravir to have a shorter hospital stay (13.29 days compared to 15.89 for CQ, _p_ = 0.06) and less likelihood of progressing to mechanical ventilation (_p_ = 0.118) or to an oxygen saturation < 90% (_p_ = 0.129).
 These results, combined with the fact that favipiravir was being compared to CQ, which is now widely understood to be ineffective for treating COVID-19, thus do not suggest that favipiravir was likely to have had a strong effect on these outcomes.
-On the other hand, another trial of 60 patients reported a significant effect of favipiravir on viral clearance at four days (a secondary endpoint), but not at 10 days (the primary endpoint) [@doi:10.1093/cid/ciaa1176].
+However, this study was retracted in November 2021 due to concerns about the data.
```

Comment:
If it was retracted for concerns about the data, I don't think we should describe it in detail.  How about

```
The second study [@doi:10.1007/s00705-021-04956-9] was retracted in November 2021 due to concerns about the data.
```
to replace these lines.

Reply:
@agitter I took this advice but moved the text around a little for flow!